app/Http/Controllers/Api/CommentController.php

```diff
@@ -77,10 +78,13 @@ public function getEpisodeComments($name, Request $request, EpisodeTransformer $
      */
     public function displayComments($episode, $episodeTransformer)
     {
-        $episode  = $episode->get();
-        $resource = new Collection($episode, $episodeTransformer);
+        $paginator = $episode->paginate(10);
```

Comment:
**Code Quality - Nitpick**
@andela-araimi Do not hardcode the limit for pagination make use of the one the user puts in the url.